crates/metadata/Cargo.toml

```diff
@@ -14,5 +14,5 @@ path = "lib.rs"
 
 [dependencies]
 serde = { version = "1.0", features = ["derive"] }
-toml = { version = "0.7", default-features = false }
```

Comment:
this was actually a bug, when you tried to compile the metadata crate on its own, you will get a compile error because the `display` feature was missing. 

When compiled together with the main project, the needed features were added because of feature unification.